examples/MicroserviceExample/WebApi/Startup.cs

```diff
@@ -40,7 +41,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSingleton<MessageSender>();
 
-            services.AddOpenTelemetryTracing((builder) => builder
+            services.AddOpenTelemetry().WithTracing((builder) => builder
```

Comment:
😄 I guess this illustrates the gotcha folks are going to run into... you're missing `.StartWithHost()`. Despite the gotcha, I am a fan of this more explicit approach.

Reply:
Woops good catch sorry! Fixed it up.